scripts/postCss.mjs

```diff
@@ -1,5 +1,10 @@
+import chalk from 'chalk';
+import { createRequire } from 'node:module';
+import * as url from 'url';
+
+const require = createRequire(import.meta.url);
+
```

Comment:
Please update file to be like the following 

https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/scripts/postCss.js